app/schema.graphql

```diff
@@ -179,6 +179,15 @@ type DimensionWithValue {
   value: String
 }
 
+type DocumentEvaluation implements Evaluation {
+  name: String!
+  score: Float
+  label: String
+  explanation: String
+  spanId: String!
```

Comment:
Did you mean to expose the spanId here? If so this becomes a bit less generic. Totally a nit.

Reply:
Good catch! I was just copy/pasting what I had in mind for the Python code.